pynvjitlink/tests/test_pynvjitlink_api.py

```diff
@@ -98,13 +104,15 @@ def test_add_cubin_with_fatbin_error(device_functions_fatbin):
         nvjitlinker.add_cubin(device_functions_fatbin, name)
 
 
+@pytest.mark.skip
```

Comment:
TODO: not sure why we need to skip this in CI but it works for me locally.

Reply:
Maybe this should be xfailed in CI instead of skipped?

Reply:
I think we still want to skip things due to https://github.com/rapidsai/pynvjitlink/pull/3